session/session.go

```diff
@@ -1726,6 +1752,17 @@ func BootstrapSession(store kv.Storage) (*domain.Domain, error) {
 		collate.EnableNewCollations()
 	}
 
+	newMemoryQuotaQuery, err := loadDefMemQuotaQuery(se)
```

Comment:
should we calculate this variable only when `config.IsMemoryQuotaQuerySetByUser` is `false`?